app/controllers/candidate_interface/references/selection_controller.rb

```diff
@@ -6,24 +6,72 @@ def new
           application_form: current_application,
           selected: current_application.application_references.selected.pluck(:id),
         )
+        @enough_references_provided = current_application.application_references.includes(:application_form).feedback_provided.count >= 2
       end
 
       def create
         @selection_form = CandidateInterface::Reference::SelectionForm.new(selection_params)
+        @enough_references_provided = current_application.application_references.includes(:application_form).feedback_provided.count >= 2
```

Comment:
As [above](https://github.com/DFE-Digital/apply-for-teacher-training/pull/4824/files#r640338586)